ui/pages/onboarding-flow/privacy-settings/privacy-settings.js

```diff
@@ -376,29 +393,105 @@ export default function PrivacySettings() {
                   </a>,
                 ])}
 
-                <Box paddingTop={2}>
-                  {currentNetwork ? (
-                    <div className="privacy-settings__network">
-                      <>
-                        <PickerNetwork
-                          label={currentNetwork?.nickname}
-                          src={currentNetwork?.rpcPrefs?.imageUrl}
-                          onClick={() => dispatch(toggleNetworkMenu())}
-                        />
-                      </>
-                    </div>
-                  ) : (
-                    <ButtonSecondary
-                      size={ButtonSecondarySize.Lg}
-                      onClick={(e) => {
-                        e.preventDefault();
-                        dispatch(showModal({ name: 'ONBOARDING_ADD_NETWORK' }));
-                      }}
+                {window.metamaskFeatureFlags?.networkMenuRedesign ? (
+                  <Box paddingTop={4}>
+                    <Box
+                      display={Display.Flex}
+                      flexDirection={FlexDirection.Column}
+                      gap={5}
                     >
-                      {t('onboardingAdvancedPrivacyNetworkButton')}
-                    </ButtonSecondary>
-                  )}
-                </Box>
+                      {[CHAIN_IDS.MAINNET, CHAIN_IDS.LINEA_MAINNET].map(
+                        (chainId) => (
+                          <Box
```

Comment:
This whole thing can be a `<Button>` for accessibility; you'd also be able to remove the `cursor: pointer` here.

Reply:
But `<button>`s in HTML are not allowed to contain divs.

![Screenshot 2024-06-11 at 10 29 33 PM](https://github.com/MetaMask/metamask-extension/assets/3500406/d069c786-fbbf-45b8-8544-2c95828c097f)

Even if they did, any of our `<Button>` components (primary, secondary, or link) introduce styling that interferes.

Picking a sub element to be the button remains accessible to tab/spacebar while propagating onClick to the parent <Box>, so this seemed like the right way.

Reply:
Keeping approach based on George's suggestion.  Everything works well accessibility wise.  

https://github.com/MetaMask/metamask-extension/assets/3500406/b9a352bf-0fa1-4591-ae65-515693337dcf